docs/concepts/trading-on-vega/data-sources.md

```diff
@@ -1,103 +1,113 @@
-* Data type (e.g. float for a price) - this must be compatible with the usage of the data source (if it is a settlement price, a numeric value would be required; for a trading termination trigger which consumes no data then any data type, etc.). Note that it is possible to have more than one “compatible” type, for instance it might be that a number could be a string or a raw numeric value in a JSON data source.
-* Data source specific details (for signed message, public key of sender; for Ethereum, contract address, method name; etc.)
+* Type of data source (e.g. `vega.builtin.timestamps`, `Open Oracle`)
+* Data type (e.g. float for a price)
+* Data source specific details
 
 Data sources must be able to emit the following data types:
+* Number - for prices or in filter comparisons
+* String - to be used to compare against in filters
+* Date/Time - to compare against in filters
+* Structured data records - such as a set of key value pairs (inputs to filters)
 
-* Number (for prices or in filter comparisons)
-* String (to be used to compare against in filters)
-* Date/Time (to compare against in filters)
-* Structured data records, such as a set of key value pairs (inputs to filters)
+## Signed message data source
+Signed message data sources are a source of off-chain price data, and can be used for settling and terminating a market. They introduce a Vega transaction that represents a data result that is validated by ensuring the signed message is provided by the Ethereum public key provided in the market’s proposal.
 
-Data sources nominated for use by Vega may refer to other data sources. For instance, one that takes a source of structured data records as input and emits only the value of a named field (e.g. to return BTCUSD_PRICE from a record containing many prices) or one that takes another data source as input and emits only data that matches a set of defined filters (e.g. to return only records with specific values in the timestamp and ticket symbol fields).
-
-## Signed message data sources
-Signed message data sources are the first external data source to be supported on Vega, and can be nominated for settling a market. They introduce a Vega transaction that represents a data result. The result is validated by ensuring the signed message is signed by one of a set of public keys provided in a market’s proposal.
-
-:::info
-There are no rewards associated with being a signed message data source, nor are there fees associated with being or using a signed message data source.
-:::
+Specifically, Signed Message Data Sources are equivalent to Posters in [Compound’s Open Price Feed](https://medium.com/compound-finance/announcing-compound-open-oracle-development-cff36f06aad3), taking signed price reports and posting them to the Vega chain. As Open Oracle reports include signatures, the data can still be verified against its source. 
 
 A signed message data source must include:
```

Comment:
We should be clear here whether we mean:

1. A signed message data source definition / specification (i.e. the data included in the market proposal)
2. The data source / feed itself (i.e. the actual data message(s))

This looks like it covers both as if they are the same thing (the pubkeys and data type are from 1 if I understand the below correctly and the ABI encoded data is from 2).